release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -74,6 +74,25 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-11-OCP-on-arm"]
+==== {product-title} on ARM
+
+{product-title} 4.11 is now supported on ARM architecture based AWS user-provisioned infrastructure and bare-metal installer-provisioned infrastructure. Instance availability and installation documentation can be found in xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
+
+The following features are supported for {product-title} on ARM:
+
+* Disconnected installation support
+* Red Hat Runtimes
+* OpenShift Pipelines
+* OpenShift Servicemesh
+* Elastic file system (EFS) for AWS
+* Local storage operator on bare metal
+* Internet small computer systems (iSCSI) for bare metal
+
+The following Operators are supported for {product-title} on ARM
+
+* Special resource operator (SRO)
```

Comment:
Should the various operators in RHACM 2.5 be listed here?

Reply:
Not sure actually, but I'll check that out, thank you! Is there a list of these operators anywhere?